malcontent.go

```diff
@@ -266,8 +266,8 @@ func main() {
 			},
 			&cli.StringFlag{
 				Name:        "ignore-tags",
-				Value:       "",
-				Usage:       "Rule tags to ignore",
+				Value:       "False-positive,None",
```

Comment:
Tags are case-sensitive and I was getting compilation errors when specifying `none` so I capitalized these.